resources/zib/zib-PainScore.xml

```diff
@@ -0,0 +1,146 @@
+  <purpose value="This Observation resource represents the Dutch [zib ('Zorginformatiebouwsteen', i.e. Health and Care Information Model) PainScore-v4.0(2020EN)](https://zibs.nl/wiki/PainScore-v4.0(2020EN))." />
+  <copyright value="Copyright and related rights waived via CC0, https://creativecommons.org/publicdomain/zero/1.0/. This does not apply to information from third parties, for example a medical terminology system. The implementer alone is responsible for identifying and obtaining any necessary licenses or authorizations to utilize third party IP in connection with the specification or otherwise." />
+  <fhirVersion value="4.0.1" />
+  <mapping>
+    <identity value="zib-painscore-v4.0-2020EN" />
+    <uri value="https://zibs.nl/wiki/PainScore-v4.0(2020EN)" />
+    <name value="zib PainScore-v4.0(2020EN)" />
+  </mapping>
+  <kind value="resource" />
+  <abstract value="true" />
+  <type value="Observation" />
+  <baseDefinition value="http://hl7.org/fhir/StructureDefinition/Observation" />
+  <derivation value="constraint" />
+  <differential>
+    <element id="Observation">
+      <path value="Observation" />
+      <short value="PainScore" />
+      <alias value="PijnScore" />
+      <constraint>
+        <key value="zib-PainScore-1" />
+        <severity value="error" />
+        <human value="maxValue of 10 if method is VAS10 or NRS or 100 if VAS100" />
+        <expression value="((method.coding.code = 'VAS10' or method.coding.code = 'NRS') and valueQuantity.value &lt;= 10) or (method.coding.code='VAS100' and valueQuantity.value &lt;= 100)" />
+        <source value="http://nictiz.nl/fhir/StructureDefinition/zib-PainScore" />
+      </constraint>
+      <constraint>
+        <key value="zib-PainScore-2" />
+        <severity value="error" />
+        <human value="minValue of 1 if method is NRS or 0 if VAS" />
+        <expression value="((method.coding.code = 'VAS10' or method.coding.code = 'VAS100') and valueQuantity.value &gt;= 0) or (method.coding.code='NRS' and valueQuantity.value &gt;= 1)" />
+        <source value="http://nictiz.nl/fhir/StructureDefinition/zib-PainScore" />
+      </constraint>
+      <mapping>
+        <identity value="zib-painscore-v4.0-2020EN" />
+        <map value="NL-CM:12.9.1" />
+        <comment value="PainScore" />
+      </mapping>
+    </element>
+    <element id="Observation.code">
+      <path value="Observation.code" />
+      <patternCodeableConcept>
+        <coding>
+          <system value="http://snomed.info/sct" />
+          <code value="425401001" />
```

Comment:
Ik denk dat het toch logischer is om hier 225908003 aan te houden, zoals in de 2017-versie ook al zo was. Deze code zit dichter bij de PainScoreValue namelijk. Zie ook de discussie in Teams.